src/Component/page/Footer/Footer.js

```diff
@@ -10,31 +10,31 @@ const Footer = () => {
     <div className="footer_container">
         <div className="container_section_1">
             < div className='footer-logo-container'>
-            <img src={Logofooter}/>
+            <img src={Logofooter} alt= "logo footer"/>
             
         </div>
         <div className='inner_container_section_1'>
         <div className='footer-div'>
-            <h3>Opening hours</h3>
+            <h1>Opening hours</h1>
             <p>Mon - Sat 11am - 12pm</p>
             <p>Sun 11am - 11pm</p>
         </div>
         <div>
-            <h3>Address</h3>
+            <h1>Address</h1>
             <p>24 Brew Lane</p>
             <p>BR3 WL Cuptown</p>
         </div>
         <div>
-            <h3>Contact</h3>
+            <h1>Contact</h1>
             <p>+44 201 998 7562</p>
             <p>hello@yellowmountain.com</p>
-             <div className='mt-2-5 text-center'><img src='/Social.png'/></div>
+             <div className='social-midia mt-2-5 text-center'><img src='/Social.png' alt= "fecebook-intagran"/></div>
```

Comment:
It's good you're putting alt text here but it should really be in the form of a sentence e.g. "Facebook and Instagram Logos" so that it makes sense when the screen reader reads it out to the user